src/Data/Functor/Bind/Class.hs

```diff
@@ -201,6 +201,7 @@ instance Apply f => Apply (Backwards f) where
 instance (Apply f, Apply g) => Apply (Compose f g) where
   Compose f <.> Compose x = Compose ((<.>) <$> f <.> x)
 
+-- | A 'Constant' is not 'Applicative' unless its 'f' is a 'Monoid', but it is an instance of 'Apply'
```

Comment:
Minor quibble: perhaps say "`A 'Constant f' is not 'Applicative' unless its 'f' is a 'Monoid'`" to be absolutely clear what `'f'` is referring to.

Similar comments apply to other Haddocks in this PR that refer to type variables.

Reply:
Done.